.github/ISSUE_TEMPLATE/4-grafana_ui_component.md

```diff
@@ -0,0 +1,39 @@
+---
+name: '@grafana/ui component request'
+about: Suggest a component for the @grafana/ui package
+labels: 'area/grafana/ui'
+---
+
+<!-- 
+By using this template you will make it easeier for us to make sure that documentation and implementation stays up to date for every component in @grafana/ui
+
+Thank you!
+-->
+
+**Why this component is needed**:
+<!-- Explain your use case -->
+___
+ - [ ] Is/could it be used enough to become a pattern?
+
+ **Where is/could it be used?**:
+ Post screenshots if you have any.
+
+___
+- [ ] It has a single use case
+- [ ] It is/could be used in multiple places
+
+**Implementation** (Checklist meant for developer)
```

Comment:
Is the "developer" here the contributing developer or the maintainer developer? I suggest making this extremely clear.